src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -73,7 +73,7 @@ def setup(self, builder: Builder):
             self.initialize_simulants, creates_columns=self.columns_created
         )
 
-        builder.event.register_listener("time_step__cleanup", self.on_time_step__prepare)
+        builder.event.register_listener("time_step__cleanup", self.on_time_step__cleanup)
```

Comment:
Yikes. Glad you caught this.

Reply:
It's not really _that_ big of a deal since it's just the name of the method. But it was definitely confusing!